pkg/search/controller.go

```diff
@@ -115,7 +106,6 @@ func (c *Controller) Start(stopCh <-chan struct{}) {
 
 	defer runtime.HandleCrash()
 
-	c.informerFactory.Start(stopCh)
```

Comment:
Just a question: why do we need to delete this line?

Reply:
karmadaInformerFactory is shared between searchController and proxyController. It's start is moved to
https://github.com/karmada-io/karmada/blob/b7554264ed07be3da20646d6e822f651415814dc/cmd/karmada-search/app/options/options.go#L108-L110

Reply:
ok, thanks for your explain~
/lgtm